helm/cluster-autoscaler-app/values.yaml

```diff
@@ -38,3 +38,10 @@ azure:
   managedIdentity: true
   # Type of deployment used on Azure
   vmType: VMSS
+
+resources:
```

Comment:
@yasn77 Using resources is our usual approach but this prevents using installation level config.

As the resources will apply to all apps using this structure that are deployed from the default catalog.